[2024-02-12T22:03:48.003+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.create_snowflake_schema manual__2024-02-12T22:03:26.211050+00:00 [queued]>
[2024-02-12T22:03:48.016+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.create_snowflake_schema manual__2024-02-12T22:03:26.211050+00:00 [queued]>
[2024-02-12T22:03:48.017+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2024-02-12T22:03:48.038+0000] {taskinstance.py:2191} INFO - Executing <Task(SnowflakeOperator): create_snowflake_schema> on 2024-02-12 22:03:26.211050+00:00
[2024-02-12T22:03:48.046+0000] {standard_task_runner.py:60} INFO - Started process 89 to run task
[2024-02-12T22:03:48.053+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'create_snowflake_schema', 'manual__2024-02-12T22:03:26.211050+00:00', '--job-id', '107', '--raw', '--subdir', 'DAGS_FOLDER/Weather_dag.py', '--cfg-path', '/tmp/tmpi9op2snw']
[2024-02-12T22:03:48.055+0000] {standard_task_runner.py:88} INFO - Job 107: Subtask create_snowflake_schema
[2024-02-12T22:03:48.136+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_dag.create_snowflake_schema manual__2024-02-12T22:03:26.211050+00:00 [running]> on host 3bd3b2605e24
[2024-02-12T22:03:48.247+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='stellar' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='create_snowflake_schema' AIRFLOW_CTX_EXECUTION_DATE='2024-02-12T22:03:26.211050+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-12T22:03:26.211050+00:00'
[2024-02-12T22:03:48.249+0000] {sql.py:276} INFO - Executing: 
                    CREATE SCHEMA IF NOT EXISTS Weather ;
            
[2024-02-12T22:03:48.268+0000] {base.py:83} INFO - Using connection ID 'conn_id_snowflake' for task execution.
[2024-02-12T22:03:48.285+0000] {base.py:83} INFO - Using connection ID 'conn_id_snowflake' for task execution.
[2024-02-12T22:03:48.287+0000] {connection.py:370} INFO - Snowflake Connector for Python Version: 3.6.0, Python Version: 3.8.18, Platform: Linux-5.15.133.1-microsoft-standard-WSL2-x86_64-with-glibc2.34
[2024-02-12T22:03:48.289+0000] {connection.py:1171} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-02-12T22:03:48.751+0000] {cursor.py:1028} INFO - query: [ALTER SESSION SET autocommit=False]
[2024-02-12T22:03:48.835+0000] {cursor.py:1041} INFO - query execution done
[2024-02-12T22:03:48.836+0000] {cursor.py:1205} INFO - Number of results in first chunk: 1
[2024-02-12T22:03:48.837+0000] {sql.py:450} INFO - Running statement: CREATE SCHEMA IF NOT EXISTS Weather ;, parameters: None
[2024-02-12T22:03:48.838+0000] {cursor.py:1028} INFO - query: [CREATE SCHEMA IF NOT EXISTS Weather ;]
[2024-02-12T22:03:48.912+0000] {cursor.py:1041} INFO - query execution done
[2024-02-12T22:03:48.914+0000] {cursor.py:1205} INFO - Number of results in first chunk: 1
[2024-02-12T22:03:48.914+0000] {sql.py:459} INFO - Rows affected: 1
[2024-02-12T22:03:48.915+0000] {snowflake.py:400} INFO - Rows affected: 1
[2024-02-12T22:03:48.916+0000] {snowflake.py:401} INFO - Snowflake query id: 01b24f4b-3200-fe52-0007-a44e0001201e
[2024-02-12T22:03:48.917+0000] {cursor.py:1028} INFO - query: [COMMIT]
[2024-02-12T22:03:49.004+0000] {cursor.py:1041} INFO - query execution done
[2024-02-12T22:03:49.006+0000] {cursor.py:1205} INFO - Number of results in first chunk: 1
[2024-02-12T22:03:49.007+0000] {connection.py:718} INFO - closed
[2024-02-12T22:03:49.067+0000] {connection.py:724} INFO - No async queries seem to be running, deleting session
[2024-02-12T22:03:49.164+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=create_snowflake_schema, execution_date=20240212T220326, start_date=20240212T220348, end_date=20240212T220349
[2024-02-12T22:03:49.229+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-12T22:03:49.274+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
